tgui-next/packages/tgui/layout.js

```diff
@@ -57,7 +57,7 @@ export class Layout extends Component {
               return;
             }
             // Bring focus back to the window on every click
-            this.contentRef.current.focus();
+            // this.contentRef.current.focus();
```

Comment:
why comment this?

Reply:
i genuinely don't remember, i need to think about it

Reply:
This refocusing conflicts with NumberInput (when you single click it to enter value). So I've implemented a different refocusing strategy, self-contained in a separate file, and called when necessary.